### PR TITLE
Changed 'i.e.' in line 17 of 'lb_dns.md' and line 35 of 'faq.md'

### PR DESCRIPTION
**Pull request by Gauravsaha-97**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ The data we gather is limited to the following:
 1. the lakeFS version currently running
 1. the OS and architecture lakeFS is running on
 1. metadata regarding the database used (version, installed extensions and parameters such as DB Timezone and work memory)
-1. periodic aggregated action counters (i.e. how many "get_object" operations occurred).
+1. periodic aggregated action counters (e.g. how many "get_object" operations occurred).
```
Comment (arielshaqed) [inline docs/faq.md:35]: This "i.e." is actually correct.
```suggestion
1. periodic aggregated action counters (i.e. how many "get_object" operations occurred).
```

File: docs/deploying/lb_dns.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ By default, lakeFS operates on port 8000, and exposes a `/_health` endpoint whic
 
 1. Your security groups should allow the load balancer to access the lakeFS server.
 1. Create a target group with a listener for port 8000.
-1. Setup TLS termination using the domain names you wish to use for both endpoints (i.e. `s3.lakefs.example.com`, `*.s3.lakefs.example.com`, `lakefs.example.com`).
+1. Setup TLS termination using the domain names you wish to use for both endpoints (e.g. `s3.lakefs.example.com`, `*.s3.lakefs.example.com`, `lakefs.example.com`).
```
Comment (arielshaqed) [inline docs/deploying/lb_dns.md:17]: Cool!

**Review by arielshaqed — COMMENTED** [review]
Thanks!

The first is a great catch, and I'd like to keep it.  The second "i.e." was actually correct.  Please revert the second change and I shall be happy to merge.

**Comment by johnnyaug** [comment]
Hey @Gauravsaha-97, welcome and thank you for your effort!
There is actually an open issue for this one, #677.
At the moment I think someone is working on it, but if you would like to work on it as well then by all means go for it!
If you do, please link the issues through the PR description, and make the replacement in all other places, as described in the issue. 
I'm here if you need any help.

File: docs/deploying/lb_dns.md (hunk 2 above)
Comment (Gauravsaha-97) [reply docs/deploying/lb_dns.md:17]: @arielshaqed I have done the necessary changes.

**Comment by johnnyaug** [comment]
Fixed by #740

**Pull request closed without merging**